src/services/postService.js

```diff
@@ -37,6 +42,37 @@ module.exports = {
     })
       .catch(err => console.log(err));
   },
+
+  getPostsOfAuthors: function(authors) {
+    const authorObjs = authors.map(name => {
+      return { name };
+    });
+    return PostModel.findAll({
+      include: [
+         { model: UserModel,
+           where: {
+             [Op.or]: authorObjs
+           }
+         }
+      ],
+    });
+  },
+
+  fetchHundredxResteemedPosts: function() {
+    const getHundredxPosts = (posts) => {
+      const permLinkAndAuthors = this.extractFromSteemitResponse(posts[0]);
+      const allHundredxResteemedPosts = permLinkAndAuthors.map(({ permLink, name }) => (
+        this.findByPermLinkAndAuthor(permLink, name)
+      ));
+
+      return Promise.all(allHundredxResteemedPosts).catch(err => console.log(err));
+    }
+    const params = [[HUNDREDX_USERNAME,0,FETCH_TOP_X_POSTS]];
+    return client.sendAsync(GET_BLOG_ENTRIES, params, getHundredxPosts);
+  },
```

Comment:
If this function is returning an async call, should the function be prefixed with `async`?
ie:
```
fetchHundredxResteemedPosts: async function() {
   // do stuff
}
```

Reply:
made these changes